packages/cozy-client/src/models/note.js

```diff
@@ -0,0 +1,8 @@
+/**
+ *
+ * @param {string} notesAppUrl URL to the Notes App (https://notes.foo.mycozy.cloud)
+ * @param {File} file io.cozy.files object
+ */
+export const generateUrlForOneNote = (notesAppUrl, file) => {
```

Comment:
I guess it could be `generateUrlForNote ` no?

Reply:
You absolutely right!